docs/_tutorials/publish-subscribe.md

```diff
@@ -3,20 +3,34 @@ layout: tutorials
 title: Publish/Subscribe
 summary: Learn how to set up pub/sub on a Solace VMR.
 icon: I_dev_P+S.svg
+links:
+    - label: HelloWorldPub.c
+      link: /blob/master/src/HelloWorldPubSub/HelloWorldPub.c
+    - label: HelloWorldPub.c
+      link: /blob/master/src/HelloWorldPubSub/HelloWorldSub.c
+    - label: os.c
+      link: /blob/master/src/HelloWorldPubSub/os.c
+    - label: os.c
+      link: /blob/master/src/HelloWorldPubSub/os.c
```

Comment:
Why is this file in here twice?

Reply:
That's a typo. There is an os.o file that should be in place of the duplicate. Good catch.